_sass/minima.scss

```diff
@@ -1,3 +1,5 @@
+// This file is unmodified but must be included so our custom syntax-highlighting partial is read
```

Comment:
@handrews I can't change what branch this is open against.  Instead of closing and opening a new PR, I just pushed my `markdownify` branch to my `master`.  I fixed this wording issue in my `bundle` branch, which is for PR #86.